webapp/components/test-run.html

```diff
@@ -49,16 +49,27 @@
 
     <div>
       <div><img src="/static/{{testRun.browser_name}}_64x64.png" /></div>
-      <div>{{testRun.browser_name}} {{testRun.browser_version}}</div>
+      <div>{{displayName(testRun.browser_name)}} {{testRun.browser_version}}</div>
       <template is="dom-if" if="{{ !isDiff(testRun.browser_name) }}">
-        <div>{{testRun.os_name}} {{testRun.os_version}}</div>
+        <div>{{displayName(testRun.os_name)}} {{testRun.os_version}}</div>
         <div class="revision">@<a href="?sha={{testRun.revision}}">{{testRun.revision}}</a></div>
         <div>{{dateFormat(testRun.created_at)}}</div>
       </template>
     </div>
   </template>
 
   <script>
+    const DISPLAY_NAMES = (() => {
+      let m = new Map();
+      m.set('chrome', 'Chrome');
+      m.set('firefox', 'Firefox');
+      m.set('edge', 'Edge');
+      m.set('linux', 'Linux');
```

Comment:
Is the omission of Safari deliberate? If so can you add a comment?

Reply:
Nope, an oversight. Added.